draft-ietf-aipref-vocab.md

```diff
@@ -289,28 +289,41 @@ The act of training general purpose AI models that have the capacity to generate
 The use of assets for Generative AI Training is a proper subset of AI Training usage.
 
 
-## Search Category {#search}
+## AI Output {#ai-output}
 
-Using one or more assets in a search application
-that directs users to the location
-from which the assets were retrieved.
+Using an asset in an AI-based system
+in the generation of outputs
+that are presented to clients of that system.
 
-Search applications can be complex
-and may serve multiple purposes.
-Only those parts of applications that direct users to the location of an asset
-are included in this category of use.
-This includes the use of titles or excerpts from assets
-that are used to help users select between multiple candidate options.
+This does not apply to any assets
+that are directly provided as inputs
+to the system by clients.
+This does not include the construction of the system, only its use.
 
-Preferences for the Search category apply to those parts of applications
-that provide search capabilities,
-regardless of what other preferences are stated.
+This includes the output of search results.
+This includes outputs that are presented to human users
+and outputs that are presented to automated clients.
 
-Parts of applications that do not direct users to the location of assets,
-such as summaries,
-are not covered by this category of use.
+The use of assets for AI Output is a proper subset of Automated Processing usage.
 
-The use of assets for Search is a proper subset of Automated Processing usage.
+
+## Search {#search}
```

Comment:
There is contention about the choice of name here.  The label was chosen in an attempt to align with expectations, but the name should match the definition.